scripts/travis/build-all-platform-binaries.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
If we go with `-tags` controlled via env var, this file is probably not necessary.

Overall, it's better to have fewer coupling between .travis.yaml and these shell scripts - if we can test conditions in travis, let's keep it there. We could also use build stages that I believe allow branch/tag/PR checks right in the yaml syntax